packages/babel-plugin-transform-class-properties/test/fixtures/nested-class/super-property-in-accessor-key/output.js

```diff
@@ -16,13 +16,12 @@ let Outer = /*#__PURE__*/function (_Hello) {
   babelHelpers.inherits(Outer, _Hello);
   var _super = babelHelpers.createSuper(Outer);
   function Outer() {
-    let _babelHelpers$get$cal, _babelHelpers$get$cal2;
+    var _computedKey;
     var _thisSuper, _this;
     babelHelpers.classCallCheck(this, Outer);
     _this = _super.call(this);
+    _computedKey = babelHelpers.get((_thisSuper = babelHelpers.assertThisInitialized(_this), babelHelpers.getPrototypeOf(Outer.prototype)), "toString", _thisSuper).call(_thisSuper);
     var _A = /*#__PURE__*/new WeakMap();
-    _babelHelpers$get$cal = babelHelpers.get((_thisSuper = babelHelpers.assertThisInitialized(_this), babelHelpers.getPrototypeOf(Outer.prototype)), "toString", _thisSuper).call(_thisSuper);
-    _babelHelpers$get$cal2 = babelHelpers.get((_thisSuper = babelHelpers.assertThisInitialized(_this), babelHelpers.getPrototypeOf(Outer.prototype)), "toString", _thisSuper).call(_thisSuper);
```

Comment:
This is a bug fix. The input is
```js
"use strict";
class Hello {
  toString() {
    return 'hello';
  }
}

class Outer extends Hello {
  constructor() {
    super();
    class Inner {
      accessor [super.toString()] = 'hello';
    }

    return new Inner();
  }
}

expect(new Outer().hello).toBe('hello');
```
As we can see from the output, the `super.toString()` is executed twice when we transform the accessor to get-set pairs since we forgot to memoize the computed key.